packages/manager/apps/telecom/src/app/telecom/telephony/alias/portability/portabilities/translations/Messages_fr_FR.json

```
@@ -43,5 +43,13 @@
   "telephony_alias_portabilities_cancel_doing": "Annulation en cours…",
   "telephony_alias_portabilities_cancel_success": "La demande de portabilité a bien été annulée.",
   "telephony_alias_portabilities_cancel_error": "Oups ! Une erreur est survenue lors de l'annulation de votre demande de portabilité.",
-  "telephony_alias_portabilities_load_error": "Oups ! Une erreur est survenue lors du chargement des portabilités en cours."
-}
\ No newline at end of file
+  "telephony_alias_portabilities_load_error": "Oups ! Une erreur est survenue lors du chargement des portabilités en cours.",
```

Weird phrase, this should be : 
```suggestion
  "telephony_alias_portabilities_load_error": "Oups ! Une erreur est survenue lors du chargement des portabilités.",
```